28.task.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Document</title>
+	<style>
+		td {	
+			text-align: center;
+			width: 20px;
+			height: 20px;
+			border: 1px solid red;	
+		}
+	</style>
+</head>
+<body>
+	<script>
+		let arr = [] 
+		for(n=1; n<=100; n++) {
+			arr.push(n)
+		}
+		
+		let table = document.createElement('table')
```

Comment:
Plz, refactor
```
const table = document.createElement("table");
let counter = 1;

for(let i = 1; i <= 10; i++) {
  const tableRow = document.createElement("tr");

  for(let j = 1; j <= 10; j++) {
    const tableCell = document.createElement("td");

    tableCell.innerText = counter++;
    // Or multiplication table instead of counter
    // tableCell.innerText = i * j;
    tableRow.append(tableCell);
  }

  table.append(tableRow);
}

document.body.append(table);
```